doc/batch-data-loading.md

```diff
@@ -0,0 +1,28 @@
+# System administration: Batch data loading
+
+The DRIVER system has provisions to programatically load data. There are two easy ways to do this. 
```

Comment:
I would add something like "Sample Python scripts are provided for both methods, but you will likely need to edit these scripts to match your data's format", just so that it's clear up-front that editing Python code will be required.